### DISCUSSION TITLE
simultaneous connection of the transmitter

### DISCUSSION THREAD

**Discussion by ATomekGH** [ESPresense/ESPresense#248] [Q&A] [answered]
I searched the forum and couldn't find an answer to my question.
Can the device tracked by ESpresense (in my case miband 5) be connected to the phone at the same time, so as to keep the possibility of receiving notifications?
Currently, when I connect miband to my phone, I lose espresense tracking.

**Comment by ATomekGH** [comment]
Hi again. None of you have a problem with detecting a MiBand or a WearOS watch while they are paired with your phones?
Is there anyone for whom this configuration works properly?

**Reply by DeekB1983** [reply]
I have a MiBand 6 and it is connected to my phone and I am tracking it successfully, what app are you using with your miband and is it set as discoverable in the app?

**Comment by Terastar-Paperclip** [comment, marked as answer]
Same root cause as [#1107](https://github.com/ESPresense/ESPresense/discussions/1107) — DT confirmed there: "Some devices only broadcast for something to find it, but once found (connected) stop broadcasting." Many wearables (Huawei Band, GT2, some MiBand variants) work this way; once your phone holds the connection the band is silent. Cross-linking for visibility.
